tests/federation_room_get_missing_events_test.go

```diff
@@ -34,6 +34,8 @@ func TestOutboundFederationIgnoresMissingEventWithBadJSONForRoomVersion6(t *test
 	srv := federation.NewServer(t, deployment,
 		federation.HandleKeyRequests(),
 		federation.HandleMakeSendJoinRequests(),
+		// Handle any transactions that the homeserver may send when connecting to another homeserver (such as presence)
+		federation.HandleTransactionRequests(nil, nil),
```

Comment:
Alternatively you can use `srv.UnexpectedRequestsAreErrors = false`

Reply:
True, though we may still want to fail on other types of erroneous requests? Not sure how stringent we want to be I suppose.

Reply:
Yeah good point.